packages/http-smoke-testing/.travis.yml

```diff
@@ -17,13 +17,14 @@ matrix:
             env: DEPENDENCIES="--prefer-lowest --prefer-stable"
         -   php: 7.1
             env: DEPENDENCIES="--prefer-lowest --prefer-stable"
+        -   php: 7.1
+            env: CHECK_STANDARDS="true"
 
 install:
     - travis_retry composer update $DEPENDENCIES
+    - if [[ "$CHECK_STANDARDS" = "true" ]]; then composer require --dev shopsys/coding-standards:dev-master; fi
 
 script:
     - php vendor/bin/parallel-lint ./src ./tests
-    - php vendor/bin/php-cs-fixer fix --config=vendor/shopsys/coding-standards/build/phpcs-fixer.php_cs --dry-run --verbose --diff ./src ./tests
-    - php vendor/bin/phpcs --standard=vendor/shopsys/coding-standards/rulesetCS.xml --extensions=php --encoding=utf-8 --tab-width=4 -sp ./src ./tests
-    - php vendor/bin/phpmd "./src,./tests" text vendor/shopsys/coding-standards/rulesetMD.xml --extensions=php
     - php vendor/bin/phpunit tests
+    - if [[ "$CHECK_STANDARDS" = "true" ]]; then php vendor/bin/ecs check --verbose ./src ./tests; fi
```

Comment:
does `[[` allow one `=` shouldnt it be doubled ? `==`

Reply:
there seems to exist some solution also with https://docs.travis-ci.com/user/customizing-the-build#fast-finishing and https://docs.travis-ci.com/user/customizing-the-build#rows-that-are-allowed-to-fail

Reply:
Both seem to work, using == in sake of consistency with monorepo-tools (72b124db4).